src/web/browser/ophan/ophan.ts

```diff
@@ -84,7 +84,7 @@ export type TestMeta = {
     products?: OphanProduct[];
 };
 
-const record = (event: {}): void => {
+export const record = (event: {}): void => {
```

Comment:
I had to export this as I wanted to call `record` directly instead of one of the abstractions built on top of it. This matches [what we do on frontend](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/ui/bannerPicker.js#L80).